client/packages/system/src/Patient/Components/PatientOptionRenderer/PatientOptionRenderer.tsx

```diff
@@ -20,13 +20,15 @@ export const getPatientOptionRenderer =
               whiteSpace: 'nowrap',
             }}
           >
-            {item.code}
+            {patient.code}
           </Typography>
         </Box>
-        <Box display="flex" flexDirection="row" gap={1} width={80}>
-          <Typography>{item.dateOfBirth}</Typography>
+        <Box display="flex" flexDirection="row" gap={1} width={100}>
+          <Typography whiteSpace={'nowrap'} color={'secondary.main'}>
```

Comment:
just as an aside - there's no need to change, this is more informational. 
you can just specify `secondary` for the colour here 

```suggestion
          <Typography whiteSpace={'nowrap'} color="secondary"}>
```